docs/source/support/troubleshooting/index.rst

```diff
@@ -0,0 +1,8 @@
+==========================================================
+Troubleshooting
+==========================================================
+
+.. toctree::
+   :maxdepth: 2
+
```

Comment:
nit: Shouldn't known issues also fall under troubleshooting? Also the only issues there are minikube related atm, do we still support it?

Reply:
> Shouldn't known issues also fall under troubleshooting?

I think it's more like *read this before you hit it*. Troubleshooting it more like *it failed, what do I do*.

> do we still support it?

I don't think it implies support, but if someone runs it, this is just a heads up

Reply:
> I don't think it implies support, but if someone runs it, this is just a heads up

I now see that `Deploying Scylla on a Kubernetes Cluster` suggests running on minikube, so we should definitely keep it  at least until that's gone.